practice_6/src/test/java/Factory/Logger.java

```diff
@@ -0,0 +1,25 @@
+package Factory;
+
+public class Logger {
+    private static Logger logger;
+
+    private Logger() {
+    }
+
+    public static Logger getInstance(){
+        if (logger == null){
+            logger = new Logger();
+        }
+        return logger;
+    }
+
+    public void msgError(){
```

Comment:
normal implementation of a logger follows standards, so the basic logger uses: 
```java
public void log(Level level,
       String msg)
```

as you are using specific levels as methods you can simplify this by using 

```java
public void error(String msg)
```